src/pages/_document.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import Document, {
+  Html,
+  Head,
+  Main,
+  NextScript,
+  DocumentContext,
+} from 'next/document';
+
+/**
+ * Represents a custom document class for Next.js.
```

Comment:
Question, why do we need to extend the next.js document class component?

Reply:
In order to customize certain HTML tags, we need to extend from next.js.
Next has default data that goes into the HTML tag. We need to extend to customize it.